clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/RegionNameUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.davinci.kafka.consumer;
+
+import com.linkedin.venice.kafka.protocol.KafkaMessageEnvelope;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+
+
+public class RegionNameUtil {
+  private static final String REGION_NAME_UNKNOWN = "UNK";
```

Comment:
Minor nitpick is: IDK if this is going to be useful in the logs? Or if we should just completely omit adding into the logs when the region is unknown. Or alternatively, spell out in full "unknown_region" or whatever?

Reply:
Yeah makes sense.  Updated the PR. Thanks!